sdk/src/openssl/openssl_trust_handler.rs

```diff
@@ -0,0 +1,240 @@
+// Copyright 2022 Adobe. All rights reserved.
```

Comment:
Not a blocker, but I'd prefer that we make this part of the `c2pa_crypto` API surface rather than duplicating the file here.